src/Composer/ScriptHandler.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ApiPlatform\Core\Composer;
+
+use Composer\Script\Event;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Finder\Finder;
+
+class ScriptHandler
+{
+    protected static $destination = '/src/Bridge/Symfony/Bundle/Resources/public/swagger-ui/dist';
```

Comment:
The logic looks bad here. Swagger UI must be installed:

* when downloading the standard edition
* when installing the bundle in an existing Symfony project

Currently, it doesn't address these use cases. IMO we should take inspiration from https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/FrameworkBundle/Command/AssetsInstallCommand.php and copy directly files in the web directory (not in the bundle).